solutions/community-deploy/build-cortx.sh

```diff
@@ -130,6 +127,7 @@ function nginx_validation() {
 }
 nginx_validation
 # clone cortx-re repository & run build.sh
+if [ -d $LOCAL_MOUNT/cortx-re ]; then rm -rf $LOCAL_MOUNT/cortx-re; fi
 git clone https://github.com/Seagate/cortx-re && pushd cortx-re/docker/cortx-deploy/
```

Comment:
We should clone cortx-re also from provided tag instead of default `main` branch

Reply:
@shailesh-vaidya, I have added support for branch/tag, but it will fail if user provided old tag where cortx-re commit was not tagged.

Reply:
We can mention in the document that support is only available from a particular tag onwards only.

Reply:
Updated